code/game/gamemodes/disaster/disaster.dm

```diff
@@ -0,0 +1,72 @@
+/datum/game_mode/disaster
+	name = "disaster"
+	config_tag = "disaster"
+	report_type = "disaster"
+	false_report_weight = 0
+	required_players = 0
+
+	announce_span = "danger"
+	announce_text = "Catastrophic disasters in-bound!"
+
+var/finished = 0
+
+/datum/game_mode/disaster/pre_setup()
+	SSevents.scheduled = 3000
+	return 1
+
+/datum/game_mode/disaster/generate_report()
+	return "Several disasters are incoming. Evacuate to Lavaland and see to it that work continues. We can't lose money over a little space-weather! While you're at it, if you could kill of the most dangerous wildlife down there, we could arrange for your extraction. Just something to think about."
+
+/datum/game_mode/disaster/announced
```

Comment:
This makes two gamemodes, rather than defining the properties after one is announced. You probably got this from the code for extended. In that case, it was intended to have two gamemodes: one that's secret (`secret_extended`) and one that's not secret, announced to be extended (`extended`). You also have put the config_tags as the same which will clash, which makes me believe you do NOT want two separate gamemodes.

You should remove the block from 20 - 23 and remove `/announced` from the rest of these procs